.github/workflows/js-ci.yml

```diff
@@ -1,11 +1,4 @@
-on:
-  push:
-    branches:
-      - master
-    tags:
-      - "*"
-  pull_request:
-  merge_group:
```

Comment:
are you sure we can remove this?
https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-a-merge-queue#triggering-merge-group-checks-with-github-actions